src/cli/Utils.cpp

```diff
@@ -167,14 +167,14 @@ namespace Utils
                 }
             }
 
-            auto conn = QObject::connect(YubiKey::instance(), &YubiKey::userInteractionRequest, [&] {
-                err << QObject::tr("Please present or touch your YubiKey to continue…") << "\n\n" << flush;
+            QObject::connect(YubiKey::instance(), &YubiKey::userInteractionRequest, [&] {
+                err << QObject::tr("Please present or touch your YubiKey to continue...") << "\n\n" << flush;
```

Comment:
Changed this because on Windows the … prints as an extended ascii character in cmd.

Reply:
Then I'd prefer not having the dots at all. We use proper ellipses everywhere.